File: ejercicios/c++/ejercicio025.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int numero;
		
	cout << "ingrese el numero a evaluar: ";
	cin >> numero;
		
	if(numero == 1 || numero == 3 || numero == 5 || numero == 7 || numero == 8 || numero == 10 || numero == 12){
		cout << "31 dias" << endl;
	}else if(numero == 2){
		cout << "28 o 29 dias" << endl;
	}else if(numero == 4 || numero == 6 || numero == 9 || numero == 11){
		cout << "30 dias" << endl;
	}else{
		cout << "numero no valido" << endl;
	}
	return 0;
}

File: ejercicios/python/ejercicio025.py
numero = int(input("ingrese el numero a evaluar: "))

if numero == 1 or numero == 3 or numero == 5 or numero == 7 or numero == 8 or numero == 10 or numero == 12:
	print("31 dias")
elif numero == 2:
	print("28 o 29 dias")
elif numero == 4 or numero == 6 or numero == 9 or numero == 11:
	print("30 dias")
else:
	print("numero no valido")
